bot/exts/events/advent_of_code/_cog.py

```diff
@@ -65,14 +65,17 @@ def __init__(self, bot: Bot):
     @tasks.loop(minutes=10.0)
     async def completer_task(self) -> None:
         """
-        Synchronise the members with the completionist role and members who has completed all 50 AoC stars.
+        Give members who have completed all 50 AoC stars completionist role.
```

Comment:
```suggestion
        Give members who have completed all 50 AoC stars the completionist role.
```
Will do when squashing